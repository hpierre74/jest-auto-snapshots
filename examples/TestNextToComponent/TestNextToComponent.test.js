import snap from 'jest-auto-snapshots';
import TestNextToComponent from './TestNextToComponent';

/**
 * jest-auto-snapshots needs to parse the actual component file
 * in order to see which props it needs. By default, it expects
 * your test file to be in a direftory next to your component. But
 * if you prefer your tests to sit right nect to your component
 * (or any other location) you can override the default with `relativePath`
 * Note that this can also be set in your jest setup file at the root level
 */

snap(TestNextToComponent, { relativePath: './' });
