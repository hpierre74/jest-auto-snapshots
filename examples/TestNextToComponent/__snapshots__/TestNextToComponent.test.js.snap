// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when passed all props 1`] = `
<div>
  jest-auto-snapshots String Fixture
</div>
`;
